Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+                                     outputs_prefix='FortiwebVM.HttpContentRoutingPolicy',
+                                     outputs_key_field='id',
+                                     outputs=parsed_data,
+                                     raw_response=response)
+    return command_results
+
+
+def geo_exception_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """List the Geo IP Exception groups.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    response = client.geo_exception_list_request()
+    parsed_data, pagination_message, formatted_response = list_response_handler(
+        client=client, response=response, data_parser=client.parser.parse_simple_name, args=args)
+    readable_output = tableToMarkdown(name='Geo exception:',
+                                      metadata=pagination_message,
+                                      t=parsed_data,
+                                      headers=['id'],
+                                      headerTransform=string_to_table_header)
+    command_results = CommandResults(readable_output=readable_output,
+                                     outputs_prefix='FortiwebVM.GeoExceptionGroup',
+                                     outputs_key_field='id',
+                                     outputs=parsed_data,
+                                     raw_response=response)
+    return command_results
+
+
+def trigger_policy_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """List the Trigger Policies.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    response = client.trigger_policy_list_request()
+    parsed_data, pagination_message, formatted_response = list_response_handler(
+        client=client, response=response, data_parser=client.parser.parse_simple_name, args=args)
+    readable_output = tableToMarkdown(name='Content Routing Policy:',
+                                      metadata=pagination_message,
+                                      t=parsed_data,
+                                      headers=['id'],
+                                      headerTransform=string_to_table_header)
+    command_results = CommandResults(readable_output=readable_output,
+                                     outputs_prefix='FortiwebVM.TriggerPolicy',
+                                     outputs_key_field='id',
+                                     outputs=parsed_data,
+                                     raw_response=response)
+    return command_results
+
+
+def certificate_intermediate_group_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
```

Comment:
if the future (not in this PR), please match the method names to the `<action>_<...>_command` structure: `list_certificate_intermediate_group_command` in this case.